### PR TITLE
Slaughter Panda

### PR DESCRIPTION
**Pull request by Aurorablade**
It still needs some work:
Adds the slaughter panda, Sprite by Skittles Modifed by Kluys.
The slaughter panda drops cheese when eating someone (not yet working..help with code)

Other additions:
Attempts to fix demon telepathy (not yet working help with code)
Custom jaunt animations for demon subtypes, just the panda for right now (not yetworking..WTF AM I DOING)

:cl: Fethas
rscadd: Fuzzy Panda death demon
bufix: fixes demon telepathy
rscadd: Support for custom demon jaunts animations
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Not sure if meme PR or...

**Comment by TheBeoni** [comment]
Give me the sprites.... BEFORE Fox/DZD or anyone else will mark this with unholy mark

File: code/game/gamemodes/miniantags/slaughter/slaughter.dm [hunk 1]
```diff
@@ -210,6 +215,22 @@
...
+	speak_emote = list("rawrs", "growls")
+	emote_hear = list("glares", "seethes")
+	attacktext = "mauls"
+
+	icon_state = "pdaemon"
+	icon_living = "pdaemon"
+	deathmessage = "fades out into a coaslace of shadows.but you swear you can still see its eyes."
+	icon_jauntdown = "pjaunt"
+	icon_jauntup = "pjauntup"
+	loot = list(/obj/item/weapon/reagent_containers/food/snacks/weirdcheesewedge)
```
Comment (Fox-McCloud) [inline code/game/gamemodes/miniantags/slaughter/slaughter.dm]: what

**Comment by HereticChurch** [comment]
As fun and cute an idea this is, I feel it'd be better left as something to be modified or created in game with admin tools.

**Comment by FreeStylaLT** [comment]
why are we adding kluys memes into the codebase

**Comment by Earthdivine** [comment]
I....kinda have to say no. We do not allow player specific stuff in the code. Plus this is a injoke no one else will kinda get.

**Comment by Aurorablade** [comment]
would be admin spawn only and anyone could use it...I mean hell i could add a carp version...FOR THE LOVE OF GOD WE HAVE A DEMON SUBTYPE THAT TICKLES PEOPLE TO DEATH :/ and i am not doing this for the meme factor at all. I normally make prs i feel people would have fun with...

**Comment by Bram961** [comment]
Hahahahaha kek, I don't mind if other coders say no to this and it gets denied since it is quite obviously about the injoke. then again it'd be admin only so I don't know how much it would matter.
I am sort of flattered that this happened though :)

**Comment by Aurorablade** [comment]
....OVer all i DO need to fix that telepathy bug anyway so if this is forced to be closed i would like input on that...

**Comment by Earthdivine** [comment]
We still took out the other admin-specific jokes that were all admin only. Kinda don't see what reason to allow this in when a Laguhter Demon with it's Icon changed can do about the same thing.

**Comment by Regen1** [comment]
gotta agree with earth on this one
:-1:

**Comment by Aurorablade** [comment]
Closing PR.
Will eventually need to make a bugfix PR for the demon whisper.

**Comment by Draconic-saint** [comment]
... do a chicken demon next

**Pull request closed without merging**